mmv1/third_party/terraform/resources/resource_apigee_sharedflow.go

```diff
@@ -0,0 +1,463 @@
+	if err != nil {
+		return err
+	}
+
+	var obj map[string]interface{}
+	log.Printf("[DEBUG] Deleting SharedFlow %q", d.Id())
+
+	// err == nil indicates that the billing_project value was found
+	if bp, err := getBillingProject(d, config); err == nil {
+		billingProject = bp
+	}
+
+	res, err := sendRequestWithTimeout(config, "DELETE", billingProject, url, userAgent, obj, d.Timeout(schema.TimeoutDelete))
+	if err != nil {
+		return handleNotFoundError(err, d, "SharedFlow")
+	}
+
+	log.Printf("[DEBUG] Finished deleting SharedFlow %q: %#v", d.Id(), res)
+	return nil
+}
+
+func resourceApigeeSharedFlowImport(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	config := meta.(*Config)
+	if err := parseImportId([]string{
+		"organizations/(?P<org_id>[^/]+)/sharedflows/(?P<name>[^/]+)",
+		"(?P<org_id>[^/]+)/(?P<name>[^/]+)",
+	}, d, config); err != nil {
+		return nil, err
+	}
+
+	// Replace import id for the resource id
+	id, err := replaceVars(d, config, "organizations/{{org_id}}/sharedflows/{{name}}")
+
+	if err != nil {
+		return nil, fmt.Errorf("Error constructing id: %s", err)
+	}
+	d.SetId(id)
+	log.Printf("[DEBUG] resourceApigeeSharedFlowImport, id=			 	%s", id)
+
+	return []*schema.ResourceData{d}, nil
+}
+
+func flattenApigeeSharedFlowMetaData(v interface{}, d *schema.ResourceData, config *Config) interface{} {
+	if v == nil {
+		return nil
+	}
+	original := v.(map[string]interface{})
+	if len(original) == 0 {
+		return nil
+	}
+	transformed := make(map[string]interface{})
+	transformed["created_at"] =
+		flattenApigeeSharedFlowMetaDataCreatedAt(original["createdAt"], d, config)
+	transformed["last_modified_at"] =
+		flattenApigeeSharedFlowMetaDataLastModifiedAt(original["lastModifiedAt"], d, config)
+	transformed["sub_type"] =
+		flattenApigeeSharedFlowMetaDataSubType(original["subType"], d, config)
+	return []interface{}{transformed}
+}
+func flattenApigeeSharedFlowMetaDataCreatedAt(v interface{}, d *schema.ResourceData, config *Config) interface{} {
+	return v
+}
+
+func flattenApigeeSharedFlowMetaDataLastModifiedAt(v interface{}, d *schema.ResourceData, config *Config) interface{} {
+	return v
+}
+
+func flattenApigeeSharedFlowMetaDataSubType(v interface{}, d *schema.ResourceData, config *Config) interface{} {
+	return v
+}
+
+func flattenApigeeSharedFlowName(v interface{}, d *schema.ResourceData, config *Config) interface{} {
+	return v
+}
+
+func flattenApigeeSharedFlowRevision(v interface{}, d *schema.ResourceData, config *Config) interface{} {
+	return v
+}
+
+func flattenApigeeSharedFlowLatestRevisionId(v interface{}, d *schema.ResourceData, config *Config) interface{} {
+	return v
+}
+
+func expandApigeeSharedFlowName(v interface{}, d TerraformResourceData, config *Config) (interface{}, error) {
+	return v, nil
+}
+
+//sendRequestRawBodyWithTimeout is derived from sendRequestWithTimeout with direct pass throught of request body
```

Comment:
```suggestion
// sendRequestRawBodyWithTimeout is derived from sendRequestWithTimeout with direct pass through of request body
```
this is giving me a formatting error locally.